tests/all/func.rs

```diff
@@ -1890,11 +1903,12 @@ fn call_wasm_getting_subtype_func_return() -> anyhow::Result<()> {
     Ok(())
 }
 
-#[test]
+#[wasmtime_test(strategies(not(Winch)))]
```

Comment:
Similar to below adding `features(simd)` here might make sense to auto-exclude Winch